tests/queries/0_stateless/03443_log_skip_indexes.reference

```diff
@@ -0,0 +1,5 @@
+[]
+['default.users.age_i']
```

Comment:
Does field `projection` in `system.query_log` also include the database name (`default`)?